docs/tutorial/part-four/index.md

```diff
@@ -78,11 +78,12 @@ Run this command in a new terminal window:
 
 ```shell
 gatsby new tutorial-part-four https://github.com/gatsbyjs/gatsby-starter-hello-world
+cd tutorial-part-four
 ```
 
 Then install some other needed dependencies at the root of the project. We'll use the Typography theme
 Kirkham + we'll try out a CSS-in-JS library
-[Glamorous](https://glamorous.rocks/). Change to your new `tutorial-part-four` directory and run:
+[Glamorous](https://glamorous.rocks/):
```

Comment:
And change this to something like?

<pre>
[Glamorous](https://glamorous.rocks/). Change to your new `tutorial-part-four` directory:

```shell
cd tutorial-part-four
```

and then run:
</pre>

Reply:
👍

Reply:
This method would work too. We should just make it consistent. Either each time they cd into a directory, either put it right after the starter install or in a new block, like @m-allanson suggests here.